### DISCUSSION TITLE
Error messages

### DISCUSSION THREAD

**Discussion by anatheka** [clouditor/clouditor#438] [Q&A] [answered]
Currently,  we use the following error messages: 
- log.Errorf() 
- fmt.Errorf()
- status.Errorf()

# Suggestion: 
**log.Errorf()** 
- error: `%v` 
- string: `%s` 

**fmt.Errorf()**
[Go fmt](https://pkg.go.dev/fmt)
- struct: 
  -  `%v` print values
  -  `%+v` print field and value
  -  `%#v` print the struct
- string:  `%s` 
- integer:  `%d`
- errors: `%w`

**status.Errorf()**
[Go gRPC status](https://pkg.go.dev/google.golang.org/grpc/status#Errorf)
same syntax as for `fmt.Errorf`, besides `%w`

@lebogg @oxisto

**Comment by oxisto** [comment, marked as answer]
I think we harmonised this more or less to the schema written above. Closing this.
